tests/unit_tests/GetVariantInfoTest.php

```diff
@@ -364,7 +364,7 @@ public static function dataProviderGetVariantInfo ()
                 'position_end' => 2,
                 'type' => 'repeat',
                 'warnings' => array(
-                    'WNOTSUPPORTED' => 'Repeat variants are currently not supported for mapping and validation.',
+                    'WNOTSUPPORTED' => 'Repeat variants are currently not supported for mapping and validation, because external tools do not recognise them yet.',
```

Comment:
Please remove this addition, as we can't align this with VV.
(idem the other places where this has been added)